# awesome-spatial-omics [![Awesome](https://awesome.re/badge-flat2.svg)](https://awesome.re)

This wants to be a community maintained repository for collecting single-cell spatial omics resources to stay updated with this fast-growing field. 

Any [contribution](https://github.com/drighelli/awesome-spatial-omics/blob/main/CONTRIBUTING.md) is really appreciated.

At the moment we distinguish these main sections:

- Spatial Omics Technologies
  - Spatial Transcriptomics
  - Spatial Multi Modal
  - Others (if any)
- Spatial Omics Methods & Tools
  - Spatial Transcriptomics Methods & Tools
  - Spatial Multi Modal Methods & Tools
  - Others (if any)

This repo is inspired to the [Mike Love's Awesome Multi Omics](https://github.com/seandavi/awesome-single-cell) and the [Sean Davis' Awesome Single Cell](https://github.com/mikelove/awesome-multi-omics) ones.

## Spatial Omics Technologies

This section is referred to all the technologies that can be collected regarding the spatial omics field, each of this subsections could collect two main families of this technologies, spot-based and molecular-based.

### Spatial Transcriptomics
|Spot-Based||||
|:-|:-|:-|:-|
|YYYY|[10x Visium](https://www.10xgenomics.com/products/spatial-gene-expression)|Allows the detection of transcripts at spatial spot level| Ref |  

|Molecular-based||||
|:-|:-|:-|:-|
|2014|[seqFISH](https://www.seqfish.com/technology)|Allows the detection of transcripts at spatial level| [Lubeck et Al.](https://www.nature.com/articles/nmeth.2892) |
|2015|[MERFISH](http://zhuang.harvard.edu/merfish.html)|Multiplexed smFISH with error-robust barcodes| [Chen et al.](https://science.sciencemag.org/content/348/6233/aaa6090) |

### Spatial MultiModal
|Year|Title|Description|Reference|
|:-:|:--|:--|:--|
|YYYY|[10x Spatial Proteomics co-detection](https://www.10xgenomics.com/products/spatial-proteomics)|Allows the co-detection of proteins and transcripts at spatial level| Ref | 
|2020|High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue| detection of transcripts and proteins on a chip-placed tissue slice|[Liu et al 2020](https://www.cell.com/cell/fulltext/S0092-8674(20)31390-8)|


### Others (if any)
|Year|Title|Description|Reference|
|:-:|:--|:--|:--|
|2021| Spatially mapped single-cell chromatin accessibility | spatial ATAC | [Thornton et al. 2021](https://www.nature.com/articles/s41467-021-21515-7)|

## Spatial Omics Methods & Tools

This section is intended not only for papers, but also for packages, methods, approaches or guidelines for Spatial Omics data analysis and integration.
Following subsection will be subsequently divided by argument while collecting them.

### Spatial Transcriptomics Methods & Tools
|Year|Title|Language|Description|Reference|
|:-:|:--|:-:|:--|:--|
|2021|[Giotto](http://spatialgiotto.rc.fas.harvard.edu/)|R|Toolbox for integrative analysis and visualization of spatial expression data|[Dries et al. 2021](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02286-2)|

### Spatial Multi Modal Methods & Tools

### Others (if any)
|Year|Title|Language|Description|Reference|
|:-:|:--|:-:|:--|:--|
|2020| [MISTy](https://saezlab.github.io/misty/)| R |Multiview Intercellular SpaTial modeling framework for knowledge extraction and analysis of highly multiplexed, spatially resolved omics data.|[Tanevski et al.](https://www.biorxiv.org/content/10.1101/2020.05.08.084145v1.full.pdf)
